Fix typos in Audio Class Terminal Types, HID Usages and Languages from Peter Samuelson <peters@p12n.org>

--- a/usb.ids
+++ b/usb.ids
@@ -23817,7 +23817,7 @@
 AT 0201  Microphone
 AT 0202  Desktop Microphone
 AT 0203  Personal Microphone
-AT 0204  Omni-directional Microphone
+AT 0204  Omni-Directional Microphone
 AT 0205  Microphone Array
 AT 0206  Processing Microphone Array
 AT 0300  Output Undefined
@@ -23832,8 +23832,8 @@
 AT 0401  Handset
 AT 0402  Headset
 AT 0403  Speakerphone, no echo reduction
-AT 0404  Echo-suppressing speakerphone
-AT 0405  Echo-canceling speakerphone
+AT 0404  Echo-Suppressing Speakerphone
+AT 0405  Echo-Canceling Speakerphone
 AT 0500  Telephony Undefined
 AT 0501  Phone line
 AT 0502  Telephone
@@ -24055,7 +24055,7 @@
 	0b9  Elevator Trim
 	0ba  Rudder
 	0bb  Throttle
-	0bc  Flight COmmunications
+	0bc  Flight Communications
 	0bd  Flare Release
 	0be  Landing Gear
 	0bf  Toe Break
@@ -24072,8 +24072,8 @@
 	0ca  Barrel Elevation
 	0cb  Drive Plane
 	0cc  Ballast
-	0cd  Bicylce Crank
-	0ce  Handle Bars
+	0cd  Bicycle Crank
+	0ce  Handlebars
 	0cf  Front Brake
 	0d0  Rear Brake
 HUT 03  VR Controls
@@ -24345,7 +24345,7 @@
 	00b  High Cut Filter
 	00c  Low Cut Filter
 	00d  Equalizer Enable
-	00e  Sound Field ON
+	00e  Sound Field On
 	00f  Surround On
 	010  Repeat
 	011  Stereo
@@ -24888,7 +24888,7 @@
 	02d  Display Status
 	02e  Stat Not Ready
 	02f  Stat Ready
-	030  Err Not a loadable Character
+	030  Err Not a Loadable Character
 	031  Err Font Data Cannot Be Read
 	032  Cursur Position Report
 	033  Row
@@ -25156,14 +25156,14 @@
 	04  Libya
 	05  Algeria
 	06  Morocco
-	07  Tunesia
+	07  Tunisia
 	08  Oman
 	09  Yemen
 	0a  Syria
 	0b  Jordan
 	0c  Lebanon
 	0d  Kuwait
-	0e  U.A.E
+	0e  U.A.E.
 	0f  Bahrain
 	10  Qatar
 L 0002  Bulgarian
@@ -25192,7 +25192,7 @@
 	06  Ireland
 	07  South Africa
 	08  Jamaica
-	09  Carribean
+	09  Caribbean
 	0a  Belize
 	0b  Trinidad
 	0c  Zimbabwe
@@ -25228,7 +25228,7 @@
 	06  Monaco
 L 000d  Hebrew
 L 000e  Hungarian
-L 000f  Idelandic
+L 000f  Icelandic
 L 0010  Italian
 	01  Italian
 	02  Swiss
@@ -25282,13 +25282,13 @@
 L 002f  Macedonian
 L 0036  Afrikaans
 L 0037  Georgian
-L 0038  Faeroese
+L 0038  Faroese
 L 0039  Hindi
 L 003e  Malay
 	01  Malaysia
 	02  Brunei Darassalam
 L 003f  Kazak
-L 0041  Awahili
+L 0041  Swahili
 L 0043  Uzbek
 	01  Latin
 	02  Cyrillic


